src/css/components/_layout.scss

```diff
@@ -124,6 +124,10 @@ body.vjs-full-window {
 // Hide disabled or unsupported controls.
 .vjs-hidden { display: none !important; }
 
+.vjs-disabled {
```

Comment:
Wondering whether `cursor: not-allowed` is overkill, since a disabled button `<button type="button" disabled>Click Me!</button>` only shows `cursor: default` instead of `cursor: pointer`?